hazelcast-build-utils/src/main/resources/hazelcast-thirdparty-template.ftl

```diff
@@ -301,6 +286,130 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 -----
 
+Creative Commons Legal Code
```

Comment:
This file should contain all permitted licenses (allowed in the root pom and in the distribution pom).